release_notes/ocp-4-4-release-notes.adoc

```diff
@@ -2110,7 +2110,27 @@ issue by deleting the Elasticsearch deployments in sequence.
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1824006[*BZ#1824006*])
 
 * {product-title} 4.4 is not shipping with a v.4.4 Metering Operator. Customers can install or continue to run the v.4.3 Metering Operator on {product-title} 4.4 clusters.
-(link:https://bugzilla.redhat.com/show_bug.cgi?id=1829035[*BZ1829035*])
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1829035[*BZ#1829035*])
+
+* When updating an {product-title} cluster from version 4.3 to 4.4, the
+`cluster-etcd-operator` sometimes fails to upgrade because it is in a degraded
+state. This is caused by an InstallerPod failure. As a workaround, you must
+force a new revision on etcd to overcome the InstallerPod failure, which clears
+the degraded status of the etcd Operator:
```

Comment:
However you refer to the operator above, probably just echo that here.  Or reword with something like:

> As a workaround, forcing a redeployment allows the operator to recover.